modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/js/addCpConfigurationEntry.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
```

Comment:
@andrea-ale-sbarra This could be detected by Brian, even though the SF seems to skip the copyright update.